### ISSUE TITLE
PHP warning on GoogleFinanceProvider

### ISSUE THREAD

**Issue by ahubers** [florianv/swap#27]
Finding this error in our logs:

```
ERROR - 2016-03-06 00:40:01 --> Severity: Warning  --> DOMDocument::loadHTML(): Unexpected end tag : div in Entity, line: 369 /var/app/current/vendor/florianv/swap/src/Provider/GoogleFinanceProvider.php 36
```

I'll take a look and submit a pull request if I can find the issue, must be some parsing discrepancy.

**Comment by ahubers** [comment]
Hmm, actually I see an attempt to supress this via 
```php
@$document->loadHTML($content); 
```

at https://github.com/florianv/swap/blob/master/src/Provider/GoogleFinanceProvider.php#L36

**Comment by florianv** [comment]
Thanks for the PR. Can you add a test please ?

**Comment by ahubers** [comment]
Sure, will do.

**Comment by ahubers** [comment]
Added a test, however it assumes that there are no php errors within the PHPUnit setUp() process...

**Comment by florianv** [comment]
Fixed in version 2.3.1. Thanks for your contribution
